templates/solutions/telco.html

```diff
@@ -0,0 +1,425 @@
+{% extends 'base_index.html' %}
```

Comment:
@mtruj013 Can you please create a folder called "telco" inside "solutions" and rename this file as "index.html"? Also it would be nice if you create a folder called "tabs" inside the "telco" folder and put all tab related files in there. It just makes it easier to find a file when you group them like that.

Reply:
Did the same for the other 2 pages in the /solutions folder for consistency